### ISSUE TITLE
Cannot remove and create Gridmanager

### ISSUE THREAD

**Issue by stephan-fischer** [neokoenig/jQuery-gridmanager#48]
Hello,

i want to remove and recreate the Gridmanager on the fly, but i have a big problem:

**Create the gridmanager**

``` javascript
            var object  = $('<div class="gridmanager"/>');

            var gm = object.gridmanager({
                remoteUrl: app.base.baseHref
            }).data('gridmanager');
```

**Remove the Gridmanager**

``` javascript
            if ($('.gridmanager').length) {
                var gm = $('.gridmanager').data('gridmanager');
                var  canvas = gm.$el.find("#" + gm.options.canvasId);
                gm.deinitCanvas();  
                gm.cleanup();
                $('.gridmanager').remove();
            }
```

And if i recreate a new gridmanager, the buttons **select column** and **add editable area** will each time multiply. Can you give me a solution for this?

Best regards
Stephan

**Comment by neokoenig** [comment]
Which version are you using?

Sounds like gm.initDefaultButtons() is being called as part of the init process, but that there's existing markup in left over from the previous iteration.

If cleanup is being called, that should remove all the markup - what are you left with once you've 'destroyed' gridmanager? Must confess, this isn't something I've tried (the dynamic add/remove aspect).

**Comment by stephan-fischer** [comment]
I'm  using the latest version available on github: /*! gridmanager - v0.3.1 - 2014-09-22
Here is a jsFiddle:

http://jsfiddle.net/cq7bb96b/
- Press Add Button
- Press Remove Button
- Press Column Add (12... dosent matter)

And now you can see, that the buttons:
**Select Column**
**Add Editable Region**

are two times available.

If you press 5 times add  - remove add - remove etc.. and than you add 12 columns you have many buttons.

**Comment by neokoenig** [comment]
Yep, definitely something odd there - think I'll need to add some sort of check on init() that no GM stuff pre-exists in scope. Will look into it. Probably want a destroy(); function or something too.

**Comment by stephan-fischer** [comment]
Yes a destroy() function would be great!
